fastapi_contrib/db/serializers.py

```diff
@@ -21,7 +21,7 @@ def clean_d(d):
 
     async def update_one(self, filter_kwargs: dict = {}):
         instance = self.Meta.model(**self.__values__)
-        await instance.update_one(filter_kwargs, self.dict())
+        await instance.update_one(filter_kwargs=filter_kwargs, **self.dict())
```

Comment:
Just a friendly reminder that we might have a problem with passing `serializer.dict()` into any model-related functions because it was designed to filter out incoming and outgoing fields from requests.